### ISSUE TITLE
Code Coverage codecov.io

### ISSUE THREAD

**Issue by andresinaka** [robbiehanson/XMPPFramework#762]
@chrisballinger what do you think of adding codecov? so we can track the % of coverage we have and be sure to have an "ok" amount of code covered for new extensions/auth/etc we push?

**Comment by chrisballinger** [comment]
Never used codecov before. Seems like a decent idea but the coverage will be super low haha.

**Comment by chrisballinger** [comment]
Or does it also do it per pull request?

**Comment by nurcahyo** [comment]
Good Idea, for alternative we can Implement our codecoverage report when Continous Integration Server implemented, and use ``` ./vendor/bin/phpunit --coverage-html``` for Html Report.
